libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/internal/TurnIconResources.kt

```diff
@@ -2,96 +2,105 @@ package com.mapbox.navigation.trip.notification.internal
-    @DrawableRes val turnIconMergeStraight: Int,
-    @DrawableRes val turnIconMergeSlightLeft: Int,
-    @DrawableRes val turnIconMergeSlightRight: Int,
-    @DrawableRes val turnIconNewNameLeft: Int,
-    @DrawableRes val turnIconNewNameRight: Int,
-    @DrawableRes val turnIconNewNameStraight: Int,
-    @DrawableRes val turnIconNewNameSharpLeft: Int,
-    @DrawableRes val turnIconNewNameSharpRight: Int,
-    @DrawableRes val turnIconNewNameSlightLeft: Int,
-    @DrawableRes val turnIconNewNameSlightRight: Int,
-    @DrawableRes val turnIconNotificationLeft: Int,
-    @DrawableRes val turnIconNotificationRight: Int,
-    @DrawableRes val turnIconNotificationStraight: Int,
-    @DrawableRes val turnIconNotificationSharpLeft: Int,
-    @DrawableRes val turnIconNotificationSharpRight: Int,
-    @DrawableRes val turnIconNotificationSlightLeft: Int,
-    @DrawableRes val turnIconNotificationSlightRight: Int,
-    @DrawableRes val turnIconOffRamp: Int,
-    @DrawableRes val turnIconOffRampLeft: Int,
-    @DrawableRes val turnIconOffRampRight: Int,
-    @DrawableRes val turnIconOffRampSlightLeft: Int,
-    @DrawableRes val turnIconOffRampSlightRight: Int,
-    @DrawableRes val turnIconOnRamp: Int,
-    @DrawableRes val turnIconOnRampLeft: Int,
-    @DrawableRes val turnIconOnRampRight: Int,
-    @DrawableRes val turnIconOnRampStraight: Int,
-    @DrawableRes val turnIconOnRampSlightLeft: Int,
-    @DrawableRes val turnIconOnRampSlightRight: Int,
-    @DrawableRes val turnIconOnRampSharpLeft: Int,
-    @DrawableRes val turnIconOnRampSharpRight: Int,
-    @DrawableRes val turnIconRamp: Int,
-    @DrawableRes val turnIconRotary: Int,
-    @DrawableRes val turnIconRotaryLeft: Int,
-    @DrawableRes val turnIconRotaryRight: Int,
-    @DrawableRes val turnIconRotaryStraight: Int,
-    @DrawableRes val turnIconRotarySlightLeft: Int,
-    @DrawableRes val turnIconRotarySlightRight: Int,
-    @DrawableRes val turnIconRotarySharpLeft: Int,
-    @DrawableRes val turnIconRotarySharpRight: Int,
-    @DrawableRes val turnIconRoundabout: Int,
-    @DrawableRes val turnIconRoundaboutLeft: Int,
-    @DrawableRes val turnIconRoundaboutRight: Int,
-    @DrawableRes val turnIconRoundaboutStraight: Int,
-    @DrawableRes val turnIconRoundaboutSlightLeft: Int,
-    @DrawableRes val turnIconRoundaboutSlightRight: Int,
-    @DrawableRes val turnIconRoundaboutSharpLeft: Int,
-    @DrawableRes val turnIconRoundaboutSharpRight: Int,
-    @DrawableRes val turnIconTurnLeft: Int,
-    @DrawableRes val turnIconTurnRight: Int,
-    @DrawableRes val turnIconTurnStraight: Int,
-    @DrawableRes val turnIconTurnSlightLeft: Int,
-    @DrawableRes val turnIconTurnSlightRight: Int,
-    @DrawableRes val turnIconTurnSharpLeft: Int,
-    @DrawableRes val turnIconTurnSharpRight: Int,
-    @DrawableRes val turnIconUturn: Int
-) {
+import com.mapbox.navigation.ui.base.model.TurnIconResources as BaseTurnIconResources
+
+class TurnIconResources private constructor(
```

Comment:
I would revert the change to rename the file from `NotificationTurnIconResources` to `TurnIconResources`, just so that it is explicitly clear that these resources are to configure turn icons for notifications and there is no confusion with 2 files having the same name

Reply:
This class is a member of `.trip.notification` package. Adding `Notification` prefix is redundant, IMHO.

Reply:
In my opinion it improves the readability since now we have 3 `TurnIconResources` in the SDK and with multiple files opened the only way you can figure out which one belongs to which SDK is by looking at the package name which usually no one cares unless they need to import it.